lib/util/template/index.ts

```diff
@@ -189,6 +183,35 @@ const compileInputProxyHandler: ProxyHandler<CompileInput> = {
   },
 };
 
+function getChildEnvFields(): string[] {
+  const childEnv = getChildEnv({});
+  return Object.keys(childEnv);
+}
+
+function getAllowedFieldsList(): Set<string> {
```

Comment:
Are separate lists `allowedTemplateFields` and `allowedTemplateFields` necessary?

I would like to refactor this and keep only one of them (`allowedTemplateFields` because it has more fields?), but I think two different lists may be here on purpose.

Reply:
yes